openedx/core/djangoapps/course_groups/migrations/rerun_0006.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ $# -eq 0 ]; then
+    echo "$0: usage: rerun_0006.sh <arguments>. At minimum, '--settings=<environment>' is expected."
+    exit 1
+fi
+
+./manage.py lms migrate course_groups 0005 --fake "$@"
+./manage.py lms migrate course_groups 0006 "$@"
```

Comment:
Don't think you want quotes on the $@ there.

On Fri, Oct 30, 2015 at 11:09 AM, Eric Fischer <notifications@github.com>
wrote:

> In openedx/core/djangoapps/course_groups/migrations/rerun_0006.sh
> <https://github.com/edx/edx-platform/pull/10295#discussion_r43512369>:
>
> > @@ -0,0 +1,8 @@
> > +#!/bin/bash
> > +if [ $# -eq 0 ]; then
> > +    echo "$0: usage: rerun_0006.sh <arguments>. At minimum, '--settings=<environment>' is expected."
> > +    exit 1
> > +fi
> > +
> > +./manage.py lms migrate course_groups 0005 --fake "$@"
> > +./manage.py lms migrate course_groups 0006 "$@"
>
> @maxrothman <https://github.com/maxrothman> Oh yeah, great point. This
> look better?
>
> edxapp@precise64:~/edx-platform$ sh openedx/core/djangoapps/course_groups/migrations/rerun_0006.sh
> openedx/core/djangoapps/course_groups/migrations/rerun_0006.sh: usage: rerun_0006.sh <arguments>. At minimum, '--settings=<environment>' is expected.
> edxapp@precise64:~/edx-platform$ sh openedx/core/djangoapps/course_groups/migrations/rerun_0006.sh --settings=devstack
>  - Soft matched migration 0005 to 0005_cohort_membership.
> Running migrations for course_groups:
>  - Migrating backwards to just after 0005_cohort_membership.
>  < course_groups:0006_cohort_membership_data_migrate
>    (faked)
>  - Soft matched migration 0006 to 0006_cohort_membership_data_migrate.
> Running migrations for course_groups:
>  - Migrating forwards to 0006_cohort_membership_data_migrate.
>  > course_groups:0006_cohort_membership_data_migrate
>  - Migration 'course_groups:0006_cohort_membership_data_migrate' is marked for no-dry-run.
>  - Loading initial data for course_groups.
> Installed 0 object(s) from 0 fixture(s)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/10295/files#r43512369>.
>